### PR TITLE
Update export doc (fixes + add torch jit)

### PR DESCRIPTION
**Pull request by araffin**
## Description

Related to https://github.com/DLR-RM/stable-baselines3/issues/836 by @Gregwar 

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) for new features and bug fixes)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation (update in the documentation)

## Checklist:

- [ ] I've read the [CONTRIBUTION](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) guide (**required**)
- [x] I have updated the changelog accordingly (**required**).
- [x] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [x] I have updated the documentation accordingly.
- [ ] I have reformatted the code using `make format` (**required**)
- [ ] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
- [ ] I have ensured `make pytest` and `make type` both pass. (**required**)
- [x] I have checked that the documentation builds using `make doc` (**required**)

Note: You can run most of the checks using `make commit-checks`.

Note: we are using a maximum length of 127 characters per line

### CODE REVIEW

File: docs/guide/export.rst [hunk 1]
```diff
@@ -49,75 +49,142 @@ For PPO, assuming a shared feature extactor.
   from stable_baselines3 import PPO
   import torch
```
Comment (qgallouedec) [inline docs/guide/export.rst:50]:
```python
import torch as th
```

File: docs/guide/export.rst [hunk 2]
```diff
@@ -49,75 +49,142 @@ For PPO, assuming a shared feature extactor.
...
+  # model = SAC("MlpPolicy", "Pendulum-v1")
+  onnxable_model = OnnxablePolicy(model.policy.actor).to("cpu")
+
+  observation_size = model.observation_space.shape
+  dummy_input = th.randn(1, *observation_size)
+
+  jit_path = "sac_traced.pt"
+
+  # Trace and optimize the module
+  traced_module = th.jit.trace(onnxable_model.eval(), dummy_input)
```
Comment (qgallouedec) [inline docs/guide/export.rst:176]: why do we need to specify `model.eval()` here, but not in the previous examples?

File: docs/guide/export.rst [hunk 3]
```diff
@@ -49,75 +49,142 @@ For PPO, assuming a shared feature extactor.
...
+  frozen_module = th.jit.freeze(traced_module)
+  frozen_module = th.jit.optimize_for_inference(frozen_module)
+  th.jit.save(frozen_module, jit_path)
+
+  ##### Load and test with onnx
+
+  import torch as th
```
Comment (qgallouedec) [inline docs/guide/export.rst:183]: Already imported

**Review by qgallouedec — CHANGES_REQUESTED** [review]

File: docs/guide/export.rst (hunk 2 above)
Comment (araffin) [reply docs/guide/export.rst:176]: it's for the "frozen" part

File: docs/guide/export.rst (hunk 3 above)
Comment (araffin) [reply docs/guide/export.rst:183]: This is meant to be a separate script (except for the observation size)

**Review by qgallouedec — APPROVED** [review]

**Pull request merged**